README.MD

```diff
@@ -1,224 +1,277 @@
+- **Resilience:** Workers that recover from downtime automatically sync with the active workers in the cluster.
+- **Load Balancing:** Search requests made to the balancer are routed to one of the workers.
+- **Automatic Data Compaction:** [Index compaction](https://manual.manticoresearch.com/Securing_and_compacting_an_index/Compacting_an_index#Compacting-an-index) is handled automatically.
+
+---
+
+## Installation
+
+### Installation from Helm Repository
 ```bash
 helm repo add manticoresearch https://helm.manticoresearch.com
-# update values.yaml if needed, see below
+# Update values.yaml if needed
 helm install manticore -n manticore --create-namespace manticoresearch/manticoresearch
 ```
 
-## Installation from GitHub (dev version)
+### Installation from GitHub (Development Version)
 ```bash
 git clone https://github.com/manticoresoftware/manticoresearch-helm
 cd manticoresearch-helm/charts/manticoresearch/
-# update values.yaml if needed, see below
+# Update values.yaml if needed
 helm install manticore -n manticore --create-namespace .
 ```
-
-If you want to install a release version use `git checkout` <release tag> or download an archive from https://github.com/manticoresoftware/manticoresearch-helm/releases .
-
-# Quick start guide:
-
-1) Export namespace name to variable for your convenience:
-   
-```bash
-export NAMESPACE=manticore
-```
-
-### Wait until it starts
-Wait until all pods are moved to running state:
-
-```bash
-kubectl --namespace $NAMESPACE get po
-```
-
-### Forward worker's port
-Forward worker's port to your local machine:
+For a specific release version, use:
 ```bash
-export WORKER_NAME=$(kubectl get pods --namespace $NAMESPACE \
--l "app.kubernetes.io/name=manticoresearch,app.kubernetes.io/instance=manticore,app.kubernetes.io/component=worker" \
--o jsonpath="{.items[0].metadata.name}")
+git checkout <release-tag>
 ```
+or download an archive from [GitHub releases](https://github.com/manticoresoftware/manticoresearch-helm/releases).
+
+---
+
+## Quick Start Guide
+
+1. **Export Namespace and Chart Name for Convenience**
+   ```bash
+   export NAMESPACE=manticore
+   export CHART_NAME=helm
+   ```
+
+2. **Wait for All Pods to Start**
+   ```bash
+   kubectl --namespace $NAMESPACE get po
+   ```
+
+3. **Forward Ports**
+
+    If you need to expose an HTTP port, you should use **9308** instead of **9306**.
+
+    - **Worker Ports:**
+      ```bash
+      kubectl --namespace $NAMESPACE port-forward service/${CHART_NAME}-manticoresearch-worker-svc 7306:9306
+      ```
+    - **Load Balancer Ports:**
+      ```bash
+      kubectl --namespace $NAMESPACE port-forward service/${CHART_NAME}-manticoresearch-balancer-svc 6306:9306
+      ```
+      
+
+4. **Connect to the Worker**
+    - **SQL:**
+      ```bash
+      mysql -h0 -P7306
+      ```
+    - **HTTP:** HTTP is stateless, so no persistent connection setup is required.
```

Comment:
May be too difficult to comprehend for an average user. It's better to say this simpler: "if you are going to use HTTP, skip this step" or smth.